app/assets/javascripts/snowcrash/modules/items_table.js.coffee

```diff
@@ -175,8 +175,10 @@ class @ItemsTable
     setTimeout(ConsoleUpdater.updateConsole, 200);
 
   storageKey: ->
-    @project ||= $('.brand').data('project') || 'ce'
-    "project.#{@project}.#{@itemName}_columns"
+    project = $('.brand').data('project') || 'ce'
+    id = $('.note-list').data('id') || ''
+    id = "#{id}." if id
+    "project.#{project}.#{id}#{@itemName}_columns"
```

Comment:
why don't we let the HTML pass the storage-key as a data attribute instead of doing the gymnastics in the JS?

Reply:
Yes, good idea. Will implement that in the `evidence tab` PR, because in there the table is identified by its id, and that will help me find the unique storage key for each table